README.md

```diff
@@ -86,20 +87,20 @@ functions:
     vpcDiscovery:
```

Comment:
`inherit basic subnet names and override basic security group names`

`basic subnet names` probably should refer to `custom.vpcDiscovery` like for the other examples